.eslintrc

```diff
@@ -10,6 +10,7 @@
    "plugins": [
    ],
    "rules": {
-    "no-console":0
+    "no-console": 0,
+    semi: [2, "always"]
```

Comment:
I think we're going to be in a tough space if we add this to the repo unfortunately, because the rest of the project won't pass linting at all